run.sh

```
@@ -0,0 +1,30 @@
+docker build -t airflow:$VERSION -f "airflow-dockerfile" .
+docker tag airflow:$VERSION airflow/airflow-ufes:$VERSION
+docker tag airflow:$VERSION airflow/airflow-ufes:latest
+
+docker network rm $NETWORK
+docker network create $NETWORK
+
+docker-compose -f airflow-compose.yaml up -d
+
+docker-compose -f database-compose.yaml up -d
+
+cd "$CURRENT_DIR"
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)